lib/tasks/report.rake

```diff
@@ -0,0 +1,29 @@
+namespace :report do
+  desc "Show non-cancelled old fiscal year proposals"
+  task old_fiscal_year: :environment do
+    ok_to_act = ENV['OK_TO_ACT'] ? true : false
```

Comment:
what about putting this logic in a small class so it is easy to test? seems like a similar situation to the annual report, which I just backfilled tests for in https://github.com/18F/C2/pull/733